package.json

```
@@ -3,6 +3,7 @@
   "version": "2.0.2",
   "description": "SASS foundation library.",
   "main": "README.md",
```

Adding it as `style` is fine, but I wonder if we could just use `main` instead. If I've understood `sass-module-importer`'s README correctly, that should also work.

Looking at it now with a fresh pair of eyes, `"main": "README.md"` looks a bit odd. @ThePeach What do you think?

---

@james-nash In a combined Javascript + CSS module, the `main` property is usually used for the Javascript entrypoint, while the `style` property is used for the CSS entrypoint.

---

Sure, but `gravy` is just a SASS package. It doesn't expose any JS. So, in this case we may as well put `main` to better use than what it's currently doing :-)

---

True. :-)

---

Hi @ringods and thank you for the contribution!
Just to confirm, are we going with the following?

```suggestion
  "main": "sass/index.scss",
```

I have no real preference myself, unless there's a specific technical reason for doing so, I also don't see why `main` shouldn't point to the actual "artefact" that is used. Everybody knows where the README is ;)

Aside from that, once merged I'll update the README to include sass-module-importer as I think it's a valuable resource. Thanks for sharing this.

---

Yes, I think we should. Will apply that change and get this merged.

---

Ah, just realised I can't as it's a PR from a forked repo, not a branch on this repo. To speed things, up I'll approve and merge this and then make the amendment locally. Hope that's Ok with you both!